### PR TITLE
The EMOTE update

### PR DESCRIPTION
**Pull request by ClownMoff**
## About The Pull Request

Ethereals and apids get sound effects, burping added, fart cooldown buffed, emote icons updated

## Why It's Good For The Game

All races should have custom sound effects, burping is funny, 20 seconds between farts is a lot and using a latex glove to represent a slap was kinda meh

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

https://cdn.discordapp.com/attachments/943311132007997490/1206004375764602890/Sin_titulo.mp4?ex=65da6e19&is=65c7f919&hm=9e40780eca69f6d97b52d03a147baf15469cb9c5c30c125e0246e08c05606fda&

</details>

The burping sound effect is from https://pixabay.com/
Licensing page: https://pixabay.com/service/license-summary/
The ethereal and apid sounds were made by @ThePooba for MonkeStation

## Changelog
:cl: Da One, Da Only, Da Real Mother FUNKIN Captain, ThePooba
add: burp sound emote, ethereal laugh and scream, apid laugh and scream. *flipoff emote 
tweak: changed the icon for slap and highfive, updated *circle icon to be in tone with the new icons
balance: buffs fart cooldown from 20 seconds to 15 seconds.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/emote.dm [hunk 1]
```diff
@@ -11,17 +11,11 @@
 
 /datum/emote/living/bow
 	key = "bow"
-	key_third_person = "bows"
+	key_third_person = "bowss"
```
Comment (PowerfulBacon) [inline code/modules/mob/living/emote.dm]: This doesn't seem right

Comment (ClownMoff) [reply code/modules/mob/living/emote.dm]: oh yeah, that, that was me testing if comitting worked hahaha

**Comment by Howluinb** [comment]
Can we *not* add burps to the game ? Farting is already obnoxious and gross enough, I don't want to have to turn game audio off so as to avoid hearing that emote spammed into my ears, please

**Comment by ClownMoff** [comment]
*burp

**Comment by CydiaLamiales** [comment]
Frankly that burp sound effect sounds rather gross and I do not think we need more gross humor like Goon's. It simply doesn't go with our "tone" of the server. Also
> 🆑 Da One, Da Only, Da Real Mother FUNKIN Captain

Write your proper name in the changelog instead of a meme

**Comment by ClownMoff** [comment]
JixSav puts ratón as a signature of his PRs I don't think this is a problem

**Comment by Howluinb** [comment]
> JixSav puts ratón as a signature of his PRs I don't think this is a problem

raton is short and looks like an actual username, yours is 9 words long lol. just call it "The Real Captain" instead

**Comment by ClownMoff** [comment]
Because it's a PR about emotes? Why would I make 2 PRs about the exact same topic?
The glove looks horrible, a yellow hand represents an emoji hand and its a neutral ground for all races, since just making it a human hand looks bad.

**Comment by ClownMoff** [comment]
The fart cooldown was an arbitrary number set by Ruko. Both burping and farting share the same cooldown, so people can't spam both

**Pull request closed without merging**